File: Java Project  V1.7/backend-spring/src/main/java/com/emart/entity/PaymentStatus.java
package com.emart.entity;

/**
 * Payment state of an Order. This project simulates the "credit
 * card verification page" the BRD describes (no real payment
 * gateway) - PAID is set synchronously once the mock Pay step
 * succeeds.
 */
public enum PaymentStatus {
    PENDING,
    PAID,
    FAILED,
    CANCELLED
}

File: DotNet Project V2.0/backend-dotnet/src/Emart.Domain/Enums/PaymentStatus.cs
namespace Emart.Domain.Enums;

/// <summary>
/// Payment state of an Order. This project simulates the "credit card verification page" the
/// BRD describes (no real payment gateway) - PAID is set synchronously once the mock Pay step succeeds.
/// </summary>
public enum PaymentStatus
{
    PENDING,
    PAID,
    FAILED,
    CANCELLED
}
